### PR TITLE
[LA-2024] Formatting pages and more

### PR DESCRIPTION
**Pull request by omargohan**
**Summary of changes:**

- Applied justified text to all pages.
- Updated sponsor information/page
- Updated contact page

**Tested:**

- Locally using hugo

### CODE REVIEW

File: data/events/2024-los-angeles.yml [hunk 1]
```diff
@@ -62,25 +62,31 @@ team_members: # Name is the only required field for team members.
...
   - name: "Murriel Perez-McCabe"
     employer: "Google"
+    github: "murriel"
+    image: "murriel.jpg"
     linkedin: "https://www.linkedin.com/in/murrielperez/"
     twitter: "xmurriel"
   - name: "Jordan Schwartz"
-  - name: "Carlos Meza"
-    github: "cmeza99"
-    linkedin: "https://www.linkedin.com/in/cmeza99"
```
Comment (adrianmoisey) [inline data/events/2024-los-angeles.yml:75]: When removing an organiser, please email info@devopsdays.org to that we can keep the mailing list and slack up to date

Comment (omargohan) [reply data/events/2024-los-angeles.yml:75]: @adrianmoisey, I have sent an email to info@devopsdays.org

Please let me know if you need anything else.

Thank you!

**Review by MikeRosTX — CHANGES_REQUESTED** [review]
Cannot merge #13191 until we get **Murriel Perez-McCabe**'s email.

**Comment by omargohan** [comment]
Email has been sent. Thank you!

**Review by MikeRosTX — APPROVED** [review]

**Pull request merged**